src/main/java/io/github/thebusybiscuit/slimefun4/api/events/ClimbingPickLaunchEvent.java

```diff
@@ -0,0 +1,84 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.Vector;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.tools.ClimbingPick;
+
+/**
+ * An {@link Event} that is called whenever a {@link Player} has
+ * used a {@link ClimbingPick} on a climbable surface.
+ *
+ * @author Linox
+ *
+ * @see ClimbingPick
+ *
+ */
+public class ClimbingPickLaunchEvent extends Event implements Cancellable {
+
+    private static final HandlerList handlers = new HandlerList();
+
+    private final Player player;
+    private final Vector velocity;
+    private final ItemStack pick;
+    private boolean cancelled;
+
+    public ClimbingPickLaunchEvent(Player player, Vector velocity, ItemStack pick) {
+        super(false);
+
+        this.player = player;
+        this.velocity = velocity;
+        this.pick = pick;
+    }
+
+    public HandlerList getHandlers() {
+        return handlers;
+    }
+
+    public static HandlerList getHandlerList() {
+        return handlers;
+    }
+
+    /**
+     * This returns the {@link Player} that used the {@link ClimbingPick}.
+     *
+     * @return The {@link Player} that used
+     */
+    public Player getPlayer() {
+        return player;
+    }
+
+    /**
+     * This returns the {@link Vector} velocity that was applied to the {@link Player}
+     * who used the {@link ClimbingPick}.
+     *
+     * @return The {@link Vector} of the applied velocity
+     */
+    public Vector getVelocity() {
+        return velocity;
+    }
```

Comment:
For Velocity, we should probably also add a setter, since that's something that may be fun to play around with.
Should also add a `Validate.notNull(v, ...)` check in that setter then though.